index.bs

```diff
@@ -1068,9 +1081,11 @@ partial interface HTMLIFrameElement {
         1. Otherwise return "<code>Disabled</code>".
     1. If |feature|'s <a>default allowlist</a> is <code>*</code>, return
       "<code>Enabled</code>".
+    1. Let |document origin| be |document|'s [=Document/origin=].
     1. If |feature|'s <a>default allowlist</a> is <code>'self'</code>, and
       |origin| is [=same origin=] with |document origin|, return
       "<code>Enabled</code>".
+    1. If |document| is a headerless document, return "<code>Enabled</code>".
```

Comment:
What about defining (somewhere else in the doc):

```
A document is a <dfn>headerless document<dfn> if any of the following conditions are true:
* Its [=Document/URL=] [=matches about:blank=]
* Its [=Document/URL=] [=matches about:srcdoc=]
* Its [=Document/URL=]'s [=scheme=] is 'data'
```

And we can add additional cases as we find them. This might also be a good candidate for export to HTML if it turns out to be generally useful.